### ISSUE TITLE
With python3 not works build

### ISSUE THREAD

**Issue by nemysis** [mgedmin/pyspacewar#5]
I try to update from 0.9.8 to 1.0.0

When I use python instead of python3 all works, build and usage.

I see in NEWS.rst

```\n
- Fixed crash on startup with Numpy >= 1.10
- Dropped support for Python 2.6.
```

Is needed perhaps

```
BuildRequires:  python3-numpy
```

Here is diff

```\n
osc diff 
-Version:        0.9.8
+Version:        1.0.0
-BuildRequires:  python-setuptools
+BuildRequires:  python3-setuptools
-Requires:       python
+Requires:       python3

 %build
-python setup.py build
+python3 setup.py build

 %install
-python setup.py install --single-version-externally-managed -O1 --root=%{buildroot}
+python3 setup.py install --single-version-externally-managed -O1 --root=%{buildroot}

 # symlink icon
 mkdir -p %{buildroot}%{python_sitelib}/%{name}/icons
-ln -s %{_datadir}//icons/hicolor/48x48/apps/%{name}.png %{buildroot}%{python_sitelib}/%{name}/icons/%{name}48.png
+ln -s %{_datadir}//icons/hicolor/48x48/apps/%{name}.png %{buildroot}%{python3_sitelib}/%{name}/icons/%{name}48.png
-%{python_sitelib}/%{name}
-%{python_sitelib}/%{name}-*.egg-info
+%{python3_sitelib}/%{name}
+%{python3_sitelib}/%{name}-*.egg-inf
```

But get with build

```\n
osc build
...
[   35s] ----- building pyspacewar.spec (user abuild)
[   35s] -----------------------------------------------------------------
[   35s] -----------------------------------------------------------------
[   35s] + exec rpmbuild -ba --define '_srcdefattr (-,root,root)' --nosignature /home/abuild/rpmbuild/SOURCES/pyspacewar.spec
[   35s] sh: /usr/bin/python: No such file or directory
[   35s] Executing(%prep): /bin/sh -e /var/tmp/rpm-tmp.hrYrki
[   35s] + umask 022
[   35s] + cd /home/abuild/rpmbuild/BUILD
[   35s] + cd /home/abuild/rpmbuild/BUILD
[   35s] + rm -rf pyspacewar-1.0.0
[   35s] + /usr/bin/gzip -dc /home/abuild/rpmbuild/SOURCES/pyspacewar-1.0.0.tar.gz
[   35s] + /bin/tar -xf -
[   35s] + STATUS=0
[   35s] + '[' 0 -ne 0 ']'
[   35s] + cd pyspacewar-1.0.0
[   35s] + /usr/bin/chmod -Rf a+rX,u+w,g-w,o-w .
[   35s] + exit 0
[   35s] Executing(%build): /bin/sh -e /var/tmp/rpm-tmp.U8yv5g
[   35s] + umask 022
[   35s] + cd /home/abuild/rpmbuild/BUILD
[   35s] + /usr/bin/rm -rf /home/abuild/rpmbuild/BUILDROOT/pyspacewar-1.0.0-0.x86_64
[   35s] ++ dirname /home/abuild/rpmbuild/BUILDROOT/pyspacewar-1.0.0-0.x86_64
[   35s] + /usr/bin/mkdir -p /home/abuild/rpmbuild/BUILDROOT
[   35s] + /usr/bin/mkdir /home/abuild/rpmbuild/BUILDROOT/pyspacewar-1.0.0-0.x86_64
[   35s] + cd pyspacewar-1.0.0
[   35s] + python3 setup.py build
[   35s] Traceback (most recent call last):
[   35s]   File "setup.py", line 22, in <module>
[   35s]     """ + '\n\n'.join(file('NEWS.rst').read().split('\n\n')[:2])
[   35s] NameError: name 'file' is not defined
[   35s] error: Bad exit status from /var/tmp/rpm-tmp.U8yv5g (%build)
[   35s] 
[   35s] 
[   35s] RPM build errors:
[   35s]     Bad exit status from /var/tmp/rpm-tmp.U8yv5g (%build)
```

Please fix this.

**Comment by mgedmin** [comment]
Pyspacewar requires Python 2.7 at the moment.

I'd love to port it to  Python 3, but I *still* don't have a PyGame that supports Python 3!  See issue #2.

*checks PyPI*

Hey, what do you know, there are manylinux wheels for Python 3 there, maybe I can actually start the porting work now?

**Comment by nemysis** [comment]
Now with this diff and Python 2.7 works good build

```\n
which python
/usr/bin/python
```

```\n
rpm -qf /usr/bin/python
python-base-2.7.12-23.1.x86_64
```

**Comment by mgedmin** [comment]
Incidentally, what's this `osc diff`/`osc build` stuff?  I've never seen it before in my life.

**Comment by mgedmin** [comment]
BTW there was a bug in my setup.py that meant the `icons/` directory was being skipped during `setup.py install`.  That's been fixed in today's 1.1.0 release.  This may affect those `ln -s` commands I see in your build description.
